odxtools/diaglayer.py

```diff
@@ -156,7 +155,7 @@ def __init__(self,
         self._services: NamedItemList[Union[DiagService, SingleEcuJob]]\
             = NamedItemList(short_name_as_id, [])
         self._communication_parameters: NamedItemList[CommunicationParameterRef]\
-            = NamedItemList(lambda cp: cp._python_name(), [])
```

Comment:
@andlaus I believe with this change, all `NamedItemList` are now using `short_name_as_id`, thus could be simplified.

Reply:
i agree.